src/pages/index.tsx

```diff
@@ -1,18 +1,23 @@
-import Image from "next/image";
+import { MainSection, NavBar } from "@/components";
+import { ThemeContext } from "@/context/context";
 import { Inter } from "next/font/google";
-import Navbar from "./navabar";
-import { useState } from "react";
-export type ColorMode = "Dark Mode" | "Light Mode";
+import React, { useState } from "react";
+
```

Comment:
imports need to follow a standard order so as to avoid circular dependencies. The order is as follows:

1. external modules like next/router, etc.
2. import alias (starts from the root with @)
3. Other imports

Reply:
Images du rendu
![Capture d’écran du 2024-06-12 15-30-34](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/093f6e5d-7719-4040-ad6b-4d09e424596d)
![Capture d’écran du 2024-06-12 15-30-04](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/f01e107d-cc46-400c-97e4-de27e3e263da)
![Capture d’écran du 2024-06-12 15-29-59](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/980379f7-7634-433f-ae50-6685a7aa6947)

Reply:
Image en Responsive
![Capture d’écran du 2024-06-12 15-32-18](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/60cbb663-0661-4374-8a6e-3860aec10840)
![Capture d’écran du 2024-06-12 15-32-21](https://github.com/Stage-Research-Center-Jake-Melvin/rest_countries_api/assets/138882910/73b1093e-470f-4017-b0f4-945ea9b263c2)